config/tikv.debug.yaml

```
@@ -0,0 +1 @@
+  log-level = "debug"
```

```suggestion
log-level = "debug"
```

I think the indent is not necessary.